web/src/socket.js

```diff
@@ -81,7 +81,7 @@ socket.on(socketMessages.action.joinChannel, msg => {
   });
 
   socket.on(socketMessages.action.waitForStart, () => {
-    hashHistory.push(`/retro/${msg.channel}`);
+    browserHistory.push(`/retro/#${msg.channel}`);
```

Comment:
Consider using `/retro/retroID` and using the history object to pull the retro ID instead of parsing the URL manually.